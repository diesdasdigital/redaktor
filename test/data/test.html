{
  "markdown": "Test\n=====\n\nThis is another **markdown** test.\n",
  "path": "test",
  "updatedAt": "2020-08-28T12:38:38.010Z",
  "createdAt": "2020-08-28T12:38:38.010Z"
}