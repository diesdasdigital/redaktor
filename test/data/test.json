{
  "default": {
    "required": {
      "title": "Test",
      "pageType": "default"
    },
    "fields": {},
    "components": []
  }
}
