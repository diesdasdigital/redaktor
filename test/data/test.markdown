Test
=====

This is another **markdown** test.
