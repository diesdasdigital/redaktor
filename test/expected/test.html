<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>Test</title>
  <meta name="viewport" content="width=device-width,initial-scale=1" />
</head>
<body>
  <pre>{
  "data": {
    "default": {
      "required": {
        "title": "Test",
        "pageType": "default"
      },
      "fields": {},
      "components": []
    }
  },
  "path": "test"
}</pre>
</body>
</html>